auth/src/main/java/com/firebase/ui/auth/ui/TaskFailureLogger.java

```diff
@@ -30,6 +31,8 @@ public TaskFailureLogger(String tag, String message) {
 
     @Override
     public void onFailure(@NonNull Exception e) {
-        Log.w(mTag, mMessage, e);
+        if (!(e instanceof FirebaseAuthUserCollisionException)) {
```

Comment:
@samtstern These exceptions produce a lot of noise (around 2-3 logs to link two accounts) and FirebaseUI now fully handles these so I opted to remove any logging associated with collisions.